sjb/config/common/test_cases/online_hibernation.yml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
I would inherit from the Origin one, you get integration with Gubernator there and parameters, etc

Reply:
ok, thanks